### DISCUSSION TITLE
Black screen white light

### DISCUSSION THREAD

**Discussion by grannymaster** [awawa-dev/HyperHDR#563] [Q&A] [answered]
Well, title says it all. If I have a fully black image on the screen, the LEDs are white color. Is there a way to fix this?
Little info:
```
Video device
MiraBox Video Capture: MiraBox (video0)
Video mode
640x480x30 MJPEG
Controller type:
ws281x
Instance:
First LED instance
Your HyperHDR version:
19.0.0.0
Latest HyperHDR version:
v19.0.0.0
```

and logs:
https://pastebin.com/LKrvGy8s

**Comment by awawa-dev** [comment]
Disable minimal backlit in the processing tab. If LEDs are still on that means the grabber doesn't report RGB(0,0,0) for the particular scene. Black means exactly RGB(0,0,0) only, not RGB(1,1,1) / RGB(3,0,0)...RGB(255,255,255) etc. Can be corrected in the processing tab also (reducing overall brightness, but that will affect all the scenes)

**Comment by grannymaster** [comment]
Do you mean this setting?
https://imgur.com/a/wQ3rO3a

I switched from Hyperion because it was always a crashing mess. When running that with the same setup, I had no issues with a black screen and white light, so I assume the hardware is sufficient. I formatted the SD card which was in Pi, burned the image, and set up the controller settings.

**Reply by awawa-dev** [reply]
Yes, this setting. HyperHDR default settings & processing are different than Hyperion (e.g. HyperHDR gamma 1.5 vs Hyperion extremely aggressive gamma value in the processing tab as side effect it can reduce some low gray values to black).

**Reply by grannymaster** [reply, marked as answer]
I managed to fix it. 

Setting "Hardware brightness control" to "-1" fixes the issue.

**Reply by Rollocko** [reply]
Facing the same issue. Where to find this setting exactly? Using HyperHDR on WebOS.

**Reply by awawa-dev** [reply]
@Rollocko webos is a completely different case because it doesn't use a grabber. Probably what you perceive as black is not the color black (0,0,0), which can turn off the LEDs when minimum backlight in processing is disabled (set to 0). For example, very close to black (2,2,2) will not turn off the LED. So it depends on the video source. take a screenshot using the HyperHDR video preview screenshot button. More precise method is to use debug light source: output colors to the file.

**Reply by Rollocko** [reply]
By the way. Why isn’t PicCap a grabber? I mean it grabs somehow the inscreen to forward it.

**Reply by awawa-dev** [reply]
it's a remote capturer (video stream transmitted by flatbuffer), you don't even have "Hardware brightness control" like in the case of USB grabbers that appeared in this topic. You can only correct the stream via LUT.

**Reply by Rollocko** [reply]
Fixed it. Colored backlight under image processing was enabled. Disabled and restarted does the trick.

**Reply by awawa-dev** [reply]
`Colored backlight` alone is not enough, just set `Backlight threshold` to 0 (and `Colored backlight` option will disappear automatically because it depends on this setting).

**Reply by Rollocko** [reply]
Did it. Thanks so much.
